frontend/public/components/utils/copy-to-clipboard.tsx

```diff
@@ -19,25 +19,33 @@ export const CopyToClipboard: React.FC<CopyToClipboardProps> = React.memo((props
   // Default to value if no visible value was specified.
   const visibleValue = _.isNil(props.visibleValue) ? props.value : props.visibleValue;
 
-  return (
-    <div className="co-copy-to-clipboard">
-      <pre className="co-pre-wrap co-copy-to-clipboard__text" data-test="copy-to-clipboard">
-        {visibleValue}
-      </pre>
+  const actions = (
+    <CodeBlockAction>
       <Tooltip content={tooltipContent} trigger="click mouseenter focus" exitDelay={1250}>
         <CTC text={props.value} onCopy={() => setCopied(true)}>
           <Button
             variant="plain"
             onMouseEnter={() => setCopied(false)}
-            className="co-copy-to-clipboard__btn pf-c-clipboard-copy__group-copy"
+            className="co-copy-to-clipboard__btn"
             type="button"
           >
             <CopyIcon />
             <span className="sr-only">{t('public~Copy to clipboard')}</span>
           </Button>
         </CTC>
       </Tooltip>
-    </div>
+    </CodeBlockAction>
+  );
+
+  return (
+    <CodeBlock actions={actions} className="co-copy-to-clipboard">
+      <CodeBlockCode
+        className="pf-c-code-block__pre co-copy-to-clipboard__text"
```

Comment:
PatternFly should allow you to add `className`s without having to re-declare the `pf-c-code-block`.  Should open an issue upstream and add a comment here as this introduces a CSS dependency.

Reply:
https://github.com/patternfly/patternfly-react/issues/7156